### DISCUSSION TITLE
Log Event "Tags"

### DISCUSSION THREAD

**Discussion by craigh33** [datalust/seq-tickets#1544] [Ideas]
Not sure if there is a solution already but thought I would raise this as an idea.

For a bit of context we work primarily with teller machines and use Seq to capture statuses and useful log events so that we can identify potential issues with either the software stack or hardware.

What we would potentially like to do is have the ability to add "Tags" or "Fields" to a log event after it has been posted to our Seq instance. For example, this would allow us to monitor software events associated with a particular hardware failure or maybe filter out false positives such as someone diagnosing a faulty bit of hardware.

Thanks,
Craig

**Comment by liammclennan** [comment]
To do things like monitoring events associated with a particular hardware failure could you use [signals](https://docs.datalust.co/docs/signals)? [Columns](https://docs.datalust.co/docs/signals#showing-properties-as-columns) might also be useful.

**Reply by craigh33** [reply]
Signals and Columns are definitely something we are looking at - thanks for the info. 

What I was getting at though would be the ability to say for example tag an event with an issue ID - something that would have to be added after its posted to Seq.

**Reply by liammclennan** [reply]
I can see how that would be useful. Possibly even adding notes against events. We will have a think about it.
